src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -210,10 +210,26 @@ private string GetConnectionString(ConnectionInfo connInfo, string databaseName)
             return ConnectionService.BuildConnectionString(connInfo.ConnectionDetails);
         }
 
-        private static string RemoveExcessWhitespace(string script)
+        public static string RemoveExcessWhitespace(string script)
         {
-            // replace all multiple spaces with single space
-            return Regex.Replace(script, " {2,}", " ");
+            if(script != null)
+            {
+                // remove leading and trailing whitespace
+                script = script.Trim();
+                // replace all multiple spaces with single space
+                script = Regex.Replace(script, " {2,}", " ");
+            }
+            return script;
+        }
+
+        public static string FormatScript(string script)
+        {
+            script = RemoveExcessWhitespace(script);
+            if(!string.IsNullOrWhiteSpace(script) && !script.Equals("null"))
+            {
+                script += "\nGO";
```

Comment:
>nGO [](start = 28, length = 3)

Is it possible scripts already have GO added to them?

Reply:
No the DacFx script generation doesn't add the GO